news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Beatmap Spotlights: May 2017"
+permalink: http://osunews.tumblr.com/post/162318442723/beatmap-spotlights-may-2017
+date: 2017-05-30 17:00:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/162318442723/beatmap-spotlights-may-2017
+---
+
+
+OrVid, Classic Pursuit, Chronoxia and Trigger reign supreme in the [Beatmap Spotlights](https://osu.ppy.sh/p/chart?ch=MONTH1705) of May 2017, and more to boot! Come check out what's in store this month, and maybe even win some osu!supporter for your trouble!
```

Comment:
``some osu!supporter`` vs ``win a osu!supporter`` is a pretty strong win for the former, I'd say. supporter is more like a state rather than an actual object, so some is more applicable here.